### DISCUSSION TITLE
How can I display some RGBA bytes as an image in Slint?

### DISCUSSION THREAD

**Discussion by bowenxuuu** [slint-ui/slint#6612] [Q&A]
For example, I have a vector containing some RGBA bytes, how can I display these bytes as an image in Slint?

**Comment by ogoffart** [comment]
What programming language are you using?
You can have a property of type `image` and assign it an image created with [`slint::Image::from_rgba8`](https://docs.rs/slint/latest/slint/struct.Image.html#method.from_rgba8) from rust, or with the [right constructor](https://docs.slint.dev/latest/docs/cpp/api/structslint_1_1Image#_CPPv4N5slint5Image5ImageE17SharedPixelBufferI10Rgba8PixelE) from C++

**Reply by bowenxuuu** [reply]
I'm using Rust. Let me try it `slint::Image::from_rgba8`.
